src/main/webapp/loginpage.xhtml

```diff
@@ -194,10 +194,11 @@
                                         </h:outputLink>
                                         <ui:fragment rendered="#{fn:contains(LoginPage.authProvider.id, 'orcid')}">
                                             <p class="help-block small" style="margin-top:1em;">
-                                                #{bundle['auth.providers.orcid.helpmessage1']}
+                                                #{bundle['']}
```

Comment:
@JayanthyChengan  You've got this little snippet `#{bundle['']}` on Line 197 left over from your previous commit to remove.